doc/nrf/ug_radio_fem.rst

```diff
@@ -1,38 +1,55 @@
 .. _ug_radio_fem:
 
-Radio front-end module (FEM) support
-####################################
+Working with RF front-end modules
+#################################
 
 .. contents::
    :local:
-   :depth: 2
+   :depth: 4
+
+An RF front-end module (FEM) is a range extender that can be used to increase the range distance and the strength of a link's connection.
+In addition to increasing range, an RF FEM also increases the robustness of the link connection.
+A more robust link leads to less packet loss, meaning less retransmissions.
+The probability of successfully receiving the first packet also increases, resulting in a more stable link latency.
+
+FEMs provide a power amplifier (PA) that increases the TX power and a low-noise amplifier (LNA) that increases the RX sensitivity.
+Some FEMs, like the nRF21540, provide a power down (PDN) control that can be used to power down the FEM internal circuits, to reduce energy consumption.
+
+For testing purposes, a FEM is usually integrated in either a development kit or a shield that you connect to a development kit.
 
 This guide describes how to add support for 2 different front-end module (FEM) implementations to your application in |NCS|.
 
-|NCS| allows you to extend the radio range of your development kit with an implementation of the front-end modules.
-Front-end modules are range extenders used for boosting the link robustness and link budget of wireless SoCs.
+Software support
+****************
+
+The |NCS| provides FEM support based on the :ref:`nrfxlib:mpsl_fem` in the :ref:`nrfxlib:mpsl` (MPSL) library.
+It also supports FEM in specific test samples like :ref:`radio_test` and :ref:`direct_test_mode` or you can use you own implementation.
+
+Using MPSL
+==========
 
-The FEM support is based on the :ref:`nrfxlib:mpsl_fem`, which is integrated into the nrfxlib's MPSL library.
-This library provides nRF21540 GPIO and Simple GPIO implementations, for 3-pin and 2-pin PA/LNA interfaces, respectively.
+The MPSL library provides an implementation for the 3-pin GPIO interface of the nRF21540 and a simplified version for FEMs with a 2-pin interface.
+To use this implementation, your application must use a protocol driver that enables the FEM feature provided by the MPSL library.
+The MPSL library provides multiprotocol support, but you can also use it in applications that require only one protocol.
+To avoid conflicts, check the protocol documentation to see if it uses FEM support provided by MPSL.
```

Comment:
```suggestion
To use this implementation, your application must use a protocol driver that enables the FEM feature.
The library provides multiprotocol support, but you can also use it in applications that require only one protocol.
To avoid conflicts, check the protocol documentation to see if the protocol uses FEM support provided by MPSL.
```